CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ When you do submit a Pull Request please make sure to update the files `CHANGELO
 
 The project adheres to a specific set of styling rules and guidelines.
 
-* [The Javascript Style Guide](https://github.com/thanpolas/Practice/blob/master/Javascript-Style-Guide.md) Based on [AirBnB's style guide](https://github.com/airbnb/javascript). Applies to both front and back end codebases.
+* [The JavaScript Style Guide](https://github.com/thanpolas/Practice/blob/master/JavaScript-Style-Guide.md) Based on [AirBnB's style guide](https://github.com/airbnb/javascript). Applies to both front and back end codebases.
```

Comment:
Fix that first before mergin: https://github.com/thanpolas/Practice/blob/master/Javascript-Style-Guide.md